server/src/matchRecipes.js

```diff
@@ -41,26 +40,26 @@ const matchQuery = (
       sugars
       servings
       matchedIngredients: contains @filter(uid(tokens)) {
-        xid
+        iname
       }
       contains {
-        xid
+        iname
       }
     }
   }`;
 };
 
 type IngredientResult = {
-  xid: string
+  iname: string
 };
 
 type MatchedRecipeResult = {
-  xid: string,
-  name: string,
+  url: string,
+  title: string,
   rating: number,
   calories: number,
-  totalFat: number,
```

Comment:
I reverted to using snake_case here because I couldn't find a way to use camelCase while keeping the schema to use snake_case.